_docs/_latest/use/server.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Server
+order: 007
+---
+
+# {{ page.title }}
+
+Forseti Server is tool to retrieve and update the server settings.
+
+## Running Forseti Server
+
+#### Getting the current server configuration
+
+```bash
+$ forseti server configuration get 
+```
+
+The command above will output the configuration that is current used by Forseti server.
+
+#### Reloading the current server configuration
+
+```bash
+$ forseti server configuration reload <PATH_TO_CONFIG_FILE> 
+```
+
+The command above will have the server reload its configuration.
+ 
+`<PATH_TO_CONFIG_FILE>` is an optional argument, If not specified, 
+the default path will be used. Note: Please specify a path that the server has
+access to (e.g. a path in  the server vm or a gcs path starts with gs://).
+
+#### Getting the current log level of the server
+
+```bash
+$ forseti server log_level get
+```
+
+The command above output the current log level of the server.
+
+#### Setting the server log level
```

Comment:
This seems like it should be in configure?

Reply:
This is more like a "tool" for you to set the server configuration in run time through CLI, I think configure should contains things like configuration files instead.